trivup/clusters/KafkaCluster.py

```diff
@@ -48,6 +49,7 @@
 #
 # See conf dict structure below.
 
+from trivup.apps.OauthbearerOIDCApp import OauthbearerOIDCApp
```

Comment:
Add it to the end of the imports, after SslApp.
I tend to order the imports in the priority, most critical first.